source/opt/dead_branch_elim_pass.cpp

```diff
@@ -118,6 +118,7 @@ bool DeadBranchElimPass::GetConstInteger(uint32_t selId, uint32_t* selVal) {
   ir::Instruction* sInst = def_use_mgr_->GetDef(selId);
   uint32_t typeId = sInst->type_id();
   ir::Instruction* typeInst = def_use_mgr_->GetDef(typeId);
```

Comment:
assert(typeInst);

Reply:
Point taken.  I changed the following test to be

if (!typeInst || (typeInst->opcode() != SpvOpTypeInt)) return false;